hollow/src/main/java/com/netflix/hollow/api/client/HollowClientUpdater.java

```diff
@@ -185,7 +185,10 @@ public StackTraceRecorder getStaleReferenceUsageStackTraceRecorder() {
     }
 
     public HollowReadStateEngine getStateEngine() {
-        return hollowDataHolder.getStateEngine();
+        if(hollowDataHolder != null) {
```

Comment:
minor - oneline this?
```
return hollowDataHolder == null ? null : hollowDataHolder.getStateEngine();

Reply:
Doh. I was too quick on the merge button.